service/src/main/java/com/epam/esm/service/dto/PageSetup.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.esm.service.dto;
+
+import lombok.Data;
+
+@Data
+public class PageSetup {
+    private static final Integer DEFAULT_PAGE = 0;
+    private static final Integer DEFAULT_SIZE = 1;
+
+    private Integer page = DEFAULT_PAGE;
+    private Integer size = DEFAULT_SIZE;
+
+    public void setPage(Integer page) {
+        this.page = page != null ? page - 1 : DEFAULT_PAGE;
```

Comment:
propose to put a little comment about '-1' since it's called 'magic' - something unclear for understanding

Reply:
Comment added